lib/strategies/simple.js

```diff
@@ -147,6 +147,10 @@ SimpleConcat.prototype = merge(SimpleConcat.prototype, {
       this.footer
     ).filter(notUndefined).join(this.separator);
 
+    if (this.footer) {
+      content += '\n';
```

Comment:
Parity with the legacy build approach: https://github.com/broccolijs/broccoli-concat/blob/351b02f99360bbe9995903784109ee7e8588274f/concat.js#L213-L216